app/helpers/form_helper.rb

```diff
@@ -192,6 +192,18 @@ def autocomplete_f(f, attr, options = {})
     end
   end
 
+  def byte_size_f(f, attr, options = {})
+    options[:class] = options[:class].to_s + ' byte_spinner'
+    options[:help_inline] ||= popover('', _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified."))
+    options[:help_block] ||= content_tag(:span, :class => 'maximum-limit hidden') do
+      content_tag(:i, '', :class => 'pficon-warning-triangle-o') +
+      content_tag(:span, ' ' + _('Specified value is higher than recommended maximum'), :class => 'error-message') +
+      f.hidden_field(attr, :class => "real-hidden-value")
```

Comment:
unless you specify `:id=> nil` this field will get the same id as the fake one.
Also, I would add this after the block so it is added even if there is a different `options[:help_block]` - i.e. `options[:help_block] += f.hidden_field....`.
You can also do without adding a class - `.find('input[type=hidden]')` below will also work.